app/src/organisms/RobotSettingsDashboard/DeviceReset.tsx

```diff
@@ -40,8 +40,10 @@ const OptionLabel = styled.label<LabelProps>`
     ${({ isSelected }) =>
       isSelected === true ? COLORS.blueEnabled : COLORS.light2};
```

Comment:
Looks like Brent commented on there not needing a border but i still see a border? 
```suggestion
      isSelected === true ? COLORS.blueEnabled : undefined};
```
or
```suggestion
      isSelected === true ? COLORS.blueEnabled : COLORS.mediumBlueEnabled};
```

Reply:
on nevermind, i must have been looking at an older commit!

Reply:
looks good now 😄